### PR TITLE
Adding NFData Instances for Line

### PR DESCRIPTION
**Pull request by sseveran**

### CODE REVIEW

**Comment by maoe** [comment]
Could we fix the build failures with GHC < 8?

**Comment by sseveran** [comment]
Let me look at it today. I know that DeriveAnyClass doesn't work correctly on some older GHCs

File: influxdb.cabal [hunk 1]
```diff
@@ -69,6 +69,7 @@ library
     , bytestring >= 0.10 && < 0.11
     , clock >= 0.7 && < 0.8
     , containers >= 0.5 && < 0.6
+    , deepseq
```
Comment (maoe) [inline influxdb.cabal]: Could we set `< 1.5` here?

File: src/Database/InfluxDB/Line.hs [hunk 2]
```diff
@@ -29,6 +32,7 @@ import qualified Data.ByteString.Lazy as L
 import qualified Data.Map.Strict as Map
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as TE
+import GHC.Generics
```
Comment (maoe) [inline src/Database/InfluxDB/Line.hs]: nitpick: Could you move this import to the end of the previous section? I organize imports in this order: modules from base, libraries, and the internal library. Each group is sorted alphabetically and qualified imports come later.

**Comment by maoe** [comment]
Would you still like this to be merged?

**Comment by maoe** [comment]
Closing this due to inactivity. Please feel free to reopen it if you like.

**Pull request closed without merging**